doc/nrf/releases/release-notes-2.3.0.rst

```diff
@@ -0,0 +1,1036 @@
+    * The specification-defined illuminance regulator (:kconfig:option:`CONFIG_BT_MESH_LIGHT_CTRL_REG_SPEC`) now selects the :kconfig:option:`CONFIG_FPU` option by default, thus it is not required to enable it explicitly in the project file.
+
+nRF9160 samples
+---------------
+
+* Added the :ref:`nidd_sample` sample that demonstrates how to use Non-IP Data Delivery (NIDD).
+
+* :ref:`modem_shell_application` sample:
+
+  * Added:
+
+    * External location service handling to test :ref:`lib_location` library functionality commonly used by applications.
+      The :ref:`lib_nrf_cloud` library is used with MQTT for location requests to the cloud.
+    * New command ``th pipeline`` for executing several MoSh commands sequentially in one thread.
+    * New command ``sleep`` for introducing wait periods in between commands when using ``th pipeline``.
+    * New command ``heap`` for printing kernel and system heap usage statistics.
+
+  * Updated:
+
+    * Timeout command-line arguments for the ``location get`` command changed from integers in milliseconds to floating-point values in seconds.
+    * Replaced deprecated LwM2M API calls with calls to new functions.
+
+* :ref:`nrf_cloud_rest_cell_pos_sample` sample:
+
+  * Added the usage of GCI search option if running modem firmware v1.3.4.
+  * Updated the sample, so that it waits for RRC idle mode before requesting neighbor cell measurements.
+
+* :ref:`lwm2m_client` sample:
+
+  * Added:
+
+    * Support for nRF7002 EK shield and Wi-Fi based location.
+    * Location events and event handlers.
+
+  * Updated:
+
+    * The sensor module has been simplified.
+      It does not use application events, filtering, or configurable periods anymore.
+    * Replaced deprecated LwM2M API calls with calls to new functions.
+    * Enabled LwM2M queue mode and updated documentation accordingly.
+    * Moved configuration options from :file:`overlay-queue.conf` to default configuration :file:`prj.conf`.
+    * Removed :file:`overlay-queue.conf`.
+    * Enabled :kconfig:option:`CONFIG_LTE_LC_TAU_PRE_WARNING_NOTIFICATIONS` configuration option.
+
+* :ref:`http_application_update_sample` sample:
+
+  * Added support for the :ref:`liblwm2m_carrier_readme` library.
+
+* :ref:`nrf_cloud_mqtt_multi_service` sample:
+
+  * Added:
+
+    * MCUboot child image files to properly access external flash on newer nRF9160DK versions.
+    * An :file:`overlay_mcuboot_ext_flash.conf` file to enable MCUboot use of external flash.
+    * Sending an alert to the cloud on boot and when a temperature limit is exceeded.
+
+  * Updated:
+
+    * The sample now uses a partition in external flash for full modem FOTA updates.
+
+* :ref:`nrf_cloud_rest_device_message` sample:
+
+  * Added sending an alert to nRF Cloud on boot.
+
+* :ref:`slm_shell_sample` sample:
+
+  * Added:
+
+    * Sample for nRF52 and nRF53 Series devices to send AT commands to nRF9160 SiP from shell.
```

Comment:
I guess the supported development kits were added, not sample as this says.

Reply:
This is actually about the new sample being added, moved it to the beginning of the section and rephrased into one bullet point.